client/src/components/ProductDetails/ImageGallery.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import css from './prodStyles.css';
```

Comment:
So I can understand, why does this need to be imported here (and in the other files)? Is this needed everywhere?

Reply:
the css file needs to be imported into each component that intends to style. I just deleted the import statements from the files not using css. 

https://github.com/webpack-contrib/css-loader

It can also be more concisely imported using this syntax "import './prodStyles.css';
" -- I just changed the various import statements.